packages/views.api.unstable/src/index.ts

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2024 Palantir Technologies, Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export type {
+  AsyncParameterValueMap,
+  ParameterConfig,
+  ParameterId,
+  ParameterValueMap,
+} from "./config.js";
+export {
+  type HostMessage,
+  type HostParametersUpdatedMessage,
+  isHostParametersUpdatedMessage,
+} from "./messages/hostMessages.js";
+export {
+  isViewEmitEventMessage,
+  isViewReadyMessage,
+  type ViewEmitEventMessage,
+  type ViewMessage,
+  type ViewReadyMessage,
+} from "./messages/viewMessages.js";
+export { META_TAG_HOST_ORIGIN } from "./metaTags.js";
+export type {
+  ArrayParameterValue,
+  BooleanParameterValue,
+  DateParameterValue,
+  NumericParameterValue,
+  ParameterValue,
+  ParameterValueType,
+  StringParameterValue,
+  TimestampParameterValue,
+} from "./parameters.js";
```

Comment:
Should we be exporting all of the array variants too? `StringArrayValue` etc

Reply:
hm good question